dev/com.ibm.ws.wssecurity_fat.wsscxf.4/fat/src/com/ibm/ws/wssecurity/fat/cxf/caller/CxfCallerUNTCBHPackageTests.java

```diff
@@ -0,0 +1,426 @@
+
+        return;
+    }
+
+    /**
+     * TestDescription:
+     *
+     * This test invokes a jax-ws cxf web service.
+     * It needs to have caller key set to sign and encrypt the SOAPBody
+     * The request is request in https.
+     * Though this test is not enforced it yet.
+     *
+     */
+
+    protected void testRoutine(
+                               String thisMethod,
+                               String callerPolicy,
+                               String testMode, // Positive, positive-1, negative or negative-1... etc
+                               String portNumber,
+                               String portNumberSecure,
+                               String strServiceName,
+                               String strServicePort,
+                               String untID,
+                               String untPassword) throws Exception {
+        testSubRoutine(
+                       thisMethod,
+                       callerPolicy,
+                       testMode, // Positive, positive-1, negative or negative-1... etc
+                       portNumber,
+                       portNumberSecure,
+                       strServiceName,
+                       strServicePort,
+                       callerUNTClientUrl,
+                       "",
+                       untID,
+                       untPassword,
+                       null);
+
+        return;
+    }
+
+    protected void testRoutine(
+                               String thisMethod,
+                               String callerPolicy,
+                               String testMode, // Positive, positive-1, negative or negative-1... etc
+                               String portNumber,
+                               String portNumberSecure,
+                               String strServiceName,
+                               String strServicePort,
+                               String untID,
+                               String untPassword,
+                               String errMsgVersion) throws Exception {
+        testSubRoutine(
+                       thisMethod,
+                       callerPolicy,
+                       testMode, // Positive, positive-1, negative or negative-1... etc
+                       portNumber,
+                       portNumberSecure,
+                       strServiceName,
+                       strServicePort,
+                       callerUNTClientUrl,
+                       "",
+                       untID,
+                       untPassword,
+                       errMsgVersion);
+
+        return;
+    }
+
+    /**
+     * TestDescription:
+     *
+     * This test invokes a jax-ws cxf web service.
+     * It needs to have caller key set to sign and encrypt the SOAPBody
+     * The request is request in https.
+     * Though this test is not enforced it yet.
+     *
+     */
+
+    protected void testBadRoutine(
```

Comment:
we don't need these extra methods here. We can clean this up later

Reply:
Yes, it can be done in the PR for cbh delete cleanup.

Reply:
done in https://github.com/OpenLiberty/open-liberty/pull/23752